---
title: "Exon Nomenclature and Classification of Transcripts database (ENACTdb): A resource for analyzing alternative splicing mediated proteome diversity"
collection: publications
category: manuscripts
permalink: /publication/2015-10-01-paper-title-number-3
excerpt: 'To associate exonic variation(s) with the protein systematically, we designed the ENACT framework for uniquely annotating exons that tracks their loci in gene architecture context with encapsulating variations in splice site(s) and amino acid coding status. Thus, ENACT provides trackable exonic variation(s) association to isoform(s) and protein features enabling the assessment of functional variation due to changes in exon composition.'
date: 2024-10-29
venue: 'Bioinformatics Advances'
paperurl: 'https://doi.org/10.1093/bioadv/vbae157'
citation: 'Verma P., Thakur D. & Pandit S.B. (2024). Exon Nomenclature and Classification of Transcripts database (ENACTdb): A resource for analyzing alternative splicing mediated proteome diversity. Bioinformatics Advances, 2024; vbae157'
---

Gene transcripts are distinguished by the composition of their exons and this different exon composition may contribute to advancing proteome complexity. Despite the availability of alternative splicing information documented in various databases, a ready association of exonic variations to the protein sequence remains a mammoth task. To associate exonic variation(s) with the protein systematically, we designed the ENACT framework for uniquely annotating exons that tracks their loci in gene architecture context with encapsulating variations in splice site(s) and amino acid coding status. As part of the ENACT annotation, predicted protein features (secondary structure/disorder/Pfam domains) are mapped to exon attributes. Thus, ENACT provides trackable exonic variation(s) association to isoform(s) and protein features enabling the assessment of functional variation due to changes in exon composition. Such analyses can be readily performed through multiple views supported by the server. The exon-centric visualizations of ENACT annotated isoforms could provide insights on functional repertoire of genes due to alternative splicing and its related processes. The database is publicly available at https://www.iscbglab.in/enactdb/. It contains protein coding genes and isoforms for Caenorhabditis elegans, Drosophila melanogaster, Danio rerio, Mus musculus, and Homo sapiens.
