---
title: "Uncovering the translatome impact of transcriptome induced diversity in eukaryotes: framework and innovative insights [preprint]"
collection: publications
category: manuscripts
permalink: /publication/2024-11-06-paper-title-number-5
excerpt: 'We analyzed isoform diversity in five representative organisms and detailed the role of AS and related processes influencing exon inclusion in imparting sequence variation for human genome. Through analyses of exonic variations in two maximally diverged isoforms of human genes, we unraveled intricate splicing patterns prevalent in coding and non-coding regions.'
date: 2024-03-08
venue: 'bioRxiv'
paperurl: 'https://www.biorxiv.org/content/10.1101/2024.03.08.584087v1'
citation: 'Verma P, Thakur D, Pandit SB. Uncovering the translatome impact of transcriptome induced diversity in eukaryotes: framework and innovative insights, bioRxiv, 2024'
---

Isoform diversity is known to enhance a geneâ€™s functional repertoire. Insights into the extent of such sequence variability generated through alternative splicing (AS), may unveil the layers of gene function/regulation. Despite studies on transcriptome diversifying processes, the impact of AS or related processes on sequence diversity still needs to be explored. Current study presents an innovative framework that centralizes exonic loci while integrating protein sequence per entity with attention to splice site variability assessment. The resulting framework enables exon (features) to be tractable, facilitating a systematic, detailed analysis of isoform diversity. We analyzed isoform diversity in five representative organisms and detailed the role of AS and related processes influencing exon inclusion in imparting sequence variation for human genome. Through analyses of exonic variations in two maximally diverged isoforms of human genes, we unraveled intricate splicing patterns prevalent in coding and non-coding regions. We observed that alternative splice sites, sequence changes, and skipping of exons are prevalent in coding exons, while the alternate first exon events are predominant in non-coding exons. Our findings offer a comprehensive understanding of isoform diversity as a function of exonic entity framework, providing valuable insights into the orchestration of exonic events in shaping the proteogenomic landscape.
